Description: Support DEB_BUILD_OPTIONS='noopt'
 This patch adds support for DEB_BUILD_OPTIONS=noopt,
 which is convenient for debugging.
Author: Michael Gold <michael@bitplane.org>
Bug-Debian: http://bugs.debian.org/737364

--- a/higan/GNUmakefile
+++ b/higan/GNUmakefile
@@ -3,7 +3,8 @@
 target := tomoko
 # console := true
 
-flags += -I. -I.. -O3
+flags += -I. -I..
+flags += $(if $(filter noopt,$(DEB_BUILD_OPTIONS)),-O0,-O3)
 objects := libco emulator audio video resource
 
 # profile-guided optimization mode
